src/main/java/racingcar/model/Car.java

```diff
@@ -1,17 +1,49 @@
 package racingcar.model;
 
-class Car {
+import java.util.Random;
+
+public class Car {
+    private static final int BOUND = 10;
+    private static final int GO_CONDITION = 4;
+    private static final int MAX_NAME_LENGTH = 5;
+
     private int score;
+    private String name;
+
+    Car(String name) {
+        if (isValidName(name)) {
+            this.name = name;
+            score = 0;
+        }
+    }
```

Comment:
```suggestion
    Car(String name) {
        isValidName(name);
        this.name = name;
        score = 0;
    }
```
이걸로 충분하지 않을까요? isValidName() 메서드가 true를 리턴하지 않아도 괜찮을 것 같네요.

Reply:
이 부분은 테스트가 필요해보이네요.